multiversx/mx-api-service
# add proxy documentation

## Proposed Changes
- Update proxy endpoints with documentation
- Add `Regular Proxy` and `Snapshotless Proxy` 

## How to test
- Make more requests to the routes in the proxy
- Check documentation
